conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/tezos/TezosDataEndpoint.scala

```diff
@@ -16,22 +17,23 @@ import scala.concurrent.{ExecutionContext, Future}
   * Platform discovery routes.
   * @param apiExecutionContext   is used to call the async operations exposed by the api service
   */
-case class TezosData(
+case class TezosDataEndpoint(
```

Comment:
trivial change: I'd expect this to be plural: TezosDataEndpoints

Reply:
Changed my mind: we need a different name for this, since TezosDataEnpoints is already the trait describing the abstract endpoints algebra.
This would generate a lot of confusion.

`TezosDataRoutes` might work?

This would imply also renaming the parent interface to `ApiDataRoutes`...

Reply:
That make sense. There are too many, very similar names for classes.